go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/rancher/rancher
 
 go 1.19
 
+replace github.com/rancher/wrangler v1.1.1 => github.com/KevinJoiner/rancher-wrangler v1.0.1-0.20230630175628-85f63862bd9b
```

Comment:
I will update this with a wrangler commit after https://github.com/rancher/wrangler/pull/310 and https://github.com/rancher/rancher/pull/41817 are merged.